external/avro/src/main/scala/org/apache/spark/sql/avro/package.scala

```diff
@@ -22,21 +22,24 @@ import org.apache.spark.annotation.Experimental
 package object avro {
 
   /**
-   * Converts a binary column of avro format into its corresponding catalyst value. The specified
-   * schema must match the read data, otherwise the behavior is undefined: it may fail or return
-   * arbitrary result.
+   * Converts a binary column of avro format into its corresponding catalyst value. If a writer's
+   * schema is provided, a different (but compatible) schema can be used for reading. If no writer's
+   * schema is provided, the specified schema must match the read data, otherwise the behavior is
+   * undefined: it may fail or return arbitrary result.
    *
    * @param data the binary column.
    * @param jsonFormatSchema the avro schema in JSON string format.
+   * @param writerJsonFormatSchema the avro schema in JSON string format used to serialize the data.
    *
    * @since 2.4.0
    */
   @Experimental
   @deprecated("Please use 'org.apache.spark.sql.avro.functions.from_avro' instead.", "3.0.0")
   def from_avro(
       data: Column,
-      jsonFormatSchema: String): Column =
-    org.apache.spark.sql.avro.functions.from_avro(data, jsonFormatSchema)
+      jsonFormatSchema: String,
```

Comment:
I guess so, I'll remove it

Reply:
we can avoid this change, since it is deprecated, can't we?